### ISSUE TITLE
dotnet publish Blazor project fails at compression

### ISSUE THREAD

**Issue by Regenhardt** [dotnet/sdk#28884]
So this project has been working for quite some time now (years actually, but it's a private project with not much movement). Now I wanted to change some minimal, unrelated things about the CI pipeline and my publish job fails.

Tried locally, exact same error.

This is what I execute in the pipeline: 

dotnet publish -c Release "TeamBuilder Website/TeamBuilder Website.csproj"

but locally I reproduced the error just executing `dotnet publish`.

Here's the I think important part of the detailed output:

```
[many more of those]
E:\Repo\Teambuilder\TeamBuilder Website\obj\Debug\net6.0\blazor.publish.boot.json
         -o
E:\Repo\Teambuilder\TeamBuilder Website\obj\Debug\net6.0\compress\3WA8sGDx.br

Unrecognized command or argument 'Website\bin\Debug\net6.0\wwwroot\_framework\blazor.webassembly.js'.
Unrecognized command or argument 'Website\obj\Debug\net6.0\compress\q91Tq+2x.br'.
Unrecognized command or argument 'Website\wwwroot\appsettings.json'.
[many more of these]
```

Looks like it can't handle the space in the path or something?

Stats:
```
PS E:\Repo\Teambuilder> dotnet --info
.NET SDK (reflecting any global.json):
 Version:   6.0.301
 Commit:    43f9b18481

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19044
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\6.0.301\

Host (useful for support):
  Version: 6.0.6
  Commit:  7cca709db2

.NET SDKs installed:
  5.0.100 [C:\Program Files\dotnet\sdk]
  5.0.300 [C:\Program Files\dotnet\sdk]
  6.0.100 [C:\Program Files\dotnet\sdk]
  6.0.301 [C:\Program Files\dotnet\sdk]
```

Full project: https://gitlab.com/Regenhardt/Teambuilder

**Comment by baronfel** [comment]
This should be fixed in 6.0.4xx and up versions of the SDK. Can you try installing those and see if your issue is resolved?

**Comment by Regenhardt** [comment]
Yep works now, thanks!
